solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -230,16 +222,17 @@ def local_train(preprocessed_dir, transforms_config_file, output_dir,
   print('Local training done.')
 
 
-def cloud_train(preprocessed_dir, transforms_config_file, output_dir,
-                staging_bucket,
+def cloud_train(preprocessed_dir, schema_file, transforms_file, output_dir,
+                model_type, staging_bucket,
```

Comment:
It's the same thing with DataFlow which requires a tmp/stage location. It feels like asking for a temp GCS dir. Personally I think it is Okay to default to output folder, but up to you.